modules/nextflow/src/main/groovy/nextflow/extension/DataflowHelper.groovy

```diff
@@ -95,7 +95,7 @@ class DataflowHelper {
         @Override
         boolean onException(final DataflowProcessor processor, final Throwable t) {
             final e = t instanceof InvocationTargetException ? t.cause : t
-            OperatorImpl.log.error("@unknown", e)
+            OperatorImpl.log.debug("@unknown", e)
```

Comment:
Would not make more sense to rely only on `session.abort` to report this error?

Reply:
Do you mean removing them?. Or moving these logs to abort? Currently, the first exception reaching abort is the one that is printed.

Reply:
I was thinking more removing, but I agree better keep as debug. It can be removed later